src/components/ProgressForm/ProgressForm.tsx

```diff
@@ -0,0 +1,22 @@
+import InputWithQuestions from './InputWithQuestions';
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+import { questions } from '../constants/ProgressUpdates';
+
+const ProgressForm = () => {
+    return (
+        <form className={styles.form}>
+            {questions.map((question) => (
```

Comment:
take this out as a constant

Reply:
what constant? didn't get you, questions is already a constant

Reply:
![image](https://user-images.githubusercontent.com/57758447/237016623-d72b3c2a-ec88-4cb1-80a7-c80c5d763617.png)
Is this what you are trying to say.

Reply:
As per the latest commit I have used useReducer and with the component having a dependency on the reducer, I can not bring it out of the component.